src/ziggurat/messaging/producer.clj

```diff
@@ -81,8 +81,29 @@
         exchange-name (prefixed-queue-name topic-entity exchange-name)]
     (publish exchange-name message)))
 
+(defn- get-channel-retry-queue-timeout-ms [topic-entity channel]
+  (-> (ziggurat-config) :stream-router topic-entity :channels channel :retry :queue-timeout-ms))
+
+(defn- channel-retries-exponential-backoff-enabled [topic-entity channel]
+  (-> (ziggurat-config) :stream-router topic-entity :channels channel :retry :exponential-backoff-enabled))
+
 (defn publish-to-channel-delay-queue [topic-entity channel message]
   (let [{:keys [exchange-name queue-timeout-ms]} (:delay (rabbitmq-config))
```

Comment:
@thediantoni  A doc-string will help here. Somewhere we need to explain the new configs which have been introduced - channel delay/exponential backoff/channel specific queue timeout. 

This doc-string can help with that.

Reply:
doc-string added @mjayprateek